helm/hpcc/templates/issuers.yaml

```diff
@@ -92,9 +92,13 @@ spec:
 {{- end -}}
 
 {{- define "hpcc.addIssuer" -}}
+ {{- $issuerKind := (hasKey .me "kind" | ternary .me.kind "") -}}
  {{- $issuerEnabled := (hasKey .me "enabled" | ternary .me.enabled true) -}}
- {{- if $issuerEnabled }}
-  {{- if (hasKey .me "spec") -}} {{- /* no spec means we are referencing an external issuer and don't need to instantiate our own */ -}}
+ {{- $issuerPredefined := (hasKey .me "predefined" | ternary .me.predefined false) -}}
```

Comment:
Adding "predefined: true" as another way of saying the issuer is defined externally only because it seems terraform may not make it easy to set something to null.  Previously setting "spec: null" was the intended way of indicating that it was an external issuer and that we shouldn't generate the k8s issuer resource.

Reply:
Not worth changing, but I think this has the same behaviour as
`{{- $issuerPredefined := .me.predefined  -}}`
when used in the if below.